include/ppx/application.h

```diff
@@ -465,7 +465,8 @@ class Application
 private:
     CommandLineParser               mCommandLineParser;
     StandardOptions                 mStandardOptions;
-    uint64_t                        mMaxFrame;
+    uint64_t                        mMaxFrames;
+    float                           mRunDuration;
```

Comment:
Would you be able to add the measurement unit (seconds I think?) either as a comment or in the name?

Reply:
Sure!